src/File.php

```diff
@@ -160,7 +163,18 @@ class File extends DataObject implements AssetContainer, Thumbnail, CMSPreviewab
     ];
 
     private static $indexes = [
-        'FileHash' => true
+        'FileHash' => true,
+        // Used when viewing files in AssetAdmin ordered by Title ASC
+        // Not used for other sort (e.g. Title DESC, Created) so if the default
+        // sort in AssetAdmin changes, we should update this index.
+        'default_asset_sort' => [
+            'type' => 'index',
+            'columns' => [
+                'ParentID',
+                'IsFolder DESC',
+                'Title',
+            ],
+        ]
```

Comment:
Each new index obviously takes up some space in the database. They become more useful the more rows you have in the table but obviously take more space as you add more rows as well.

I've decided to be conservative for now and just cover the default case (similar to how we usually cover `default_sort` config with indexes, but don't cover all `summary_fields` even though gridfields usually let you sort by those).

Reply:
That said, this index _might_ also be used when sorting by Title DESC or by Created, because the `ParentID` and `IsFolder DESC` parts of the index might still be useful. That will depend on a bunch of factors so while that wasn't the case in my testing, it may happen for some projects.